scala-mode-syntax.el

```diff
@@ -600,6 +600,16 @@ symbol constituents (syntax 3)."
                  (scala-syntax:put-syntax-table-property 0 '(3 . nil)))
                '(3 . nil))))))))) ;; symbol constituent syntax (3) also for the '_'
 
+(defun scala-syntax:propertize-special-symbols (start end)
+  (save-excursion
+     (goto-char start)
+     (while (re-search-forward (concat "[" scala-syntax:opchar-group "]" scala-syntax:op-re) end t)
+       (let ((match-beg (match-beginning 0))
+             (match-end (match-end 0))
+             (match (match-string 0)))
+         (unless (member match '("/*" "//" "/**" "</" "*/"))
```

Comment:
there is negative effect in when there are no whitespaces or other delimiters as in:
```scala
class X[T<:Mapper[T]](t: T){
  //
}
```
whole `T<:Mapper` will be considered as one symbol, this as also true for all other usages of operator characters identifiers without putting whitespaces etc. around them.

With spaces as delimiters as in:
```scala
class X[T <: Mapper[T]](t: T){
  //
}
```

`T`, `<:` and `Mapper` are now three different symbols.